tests/chainer_tests/functions_tests/connection_tests/test_n_step_lstm.py

```diff
@@ -20,7 +20,7 @@ def _split(inputs, pos):
 
 
 @testing.parameterize(*testing.product({
-    'use_cudnn': [True, False],
+    'use_cudnn': ['always', 'never'],
```

Comment:
I think we do not need to test `auto` if the test only checks if it runs correctly with and without cuDNN. It reduces the number of test combinations in many cases. Of course, we should still try `auto` if the test checks if the correct mode is chosen with a given `use_cudnn` flag.